OpenTabletDriver.Daemon/DriverDaemon.cs

```diff
@@ -417,21 +417,17 @@ public Task SetTabletDebug(bool enabled)
 
         public Task<string> RequestDeviceString(int index)
         {
-            return Task.FromResult(Driver.TabletReader?.Device?.GetDeviceString(index) ?? null);
+            if (Driver.TabletReader?.Device != null)
+                return Task.FromResult(Driver.TabletReader?.Device?.GetDeviceString(index));
```

Comment:
```suggestion
                return Task.FromResult(Driver.TabletReader.Device.GetDeviceString(index));
```

We know these fields aren't null due to the guard